src/ctrv_model.cpp

```diff
@@ -75,4 +75,14 @@ auto nextState(const CtrvState& state, units::time::second_t time_step, const Ct
   return next_state;
 }
 
+void debugPrint(const CtrvState& state)
+{
+  std::cout << "CtrvState: \n";
+  std::cout << "x: " << state.position_x << std::endl;
+  std::cout << "y: " << state.position_y << std::endl;
+  std::cout << "velocity: " << state.velocity << std::endl;
+  std::cout << "yaw: " << state.yaw.get_angle() << std::endl;
+  std::cout << "yaw rate: " << state.yaw_rate << std::endl;
```

Comment:
The recommended practice is to use `'\n'` instead of `std::endl` because the latter does more than print a new line. It is a small optimization, but it is considered the "better way."